### DISCUSSION TITLE
Classic Wooden Coaster

### DISCUSSION THREAD

**Discussion by StrikerStorm** [OpenRCT2/OpenRCT2#17445] [Feature requests] [answered]
I was wondering if it is possible (either technically or legally) to add a 'Classic Wooden Coaster' in RCT1 scenarios where it is applicable? Complete with track restrictions, and 30° turn banking.

It's not too practical of a request, but I've always enjoyed the shallower banking, and more hollow sounding ride noise of the RCT1 Wooden Coaster. While I've always seen the RCT2 Wooden Coaster as the proper 'Wooden Twister Coaster' (Not including the Hybrid Coaster).

**Comment by ocalhoun6** [comment]
If the RCT1 assets are linked, maybe it actually wouldn't be _that_ hard to import it as a completely separate ride type?

I don't think that would help with it being included in RCT1 scenarios unless you manually edit the scenarios and replace the ride availability (and even demolish and rebuild preexisting wooden coasters in some scenarios). But at least you'd be able to add that ride type manually through the object selection and then enjoy building it in scenarios.

(Maybe a plugin could be developed to automatically replace RCT2 wooden coasters with RCT1 wooden coasters in RCT1 scenarios?)

**Comment by 733737** [comment]
Maybe you could recover the alternate sound effect and play it on regular wooden trains, reserve the rct2 sound effect for the articulated trains

**Comment by spacek531** [comment]
It's a known issue, there's no timeline for its development though.

**Comment by ZeeMaji** [comment]
This has now been implemented as of the latest development builds, the trains for it still need to be implemented at the objects repo but the classic wooden coaster ride type with shallow banking is now used when importing rct1 files.

**Comment by ZeeMaji** [comment, marked as answer]
The trains have been added to the objects repo and should be in develop builds once the next objects release arrives.

**Reply by StrikerStorm** [reply]
Excellent! I was following the addition of the Classic Wooden Coaster for some time; glad it would make it in! Along with the original Steel Coaster Trains, and 2-Wide Inverted Trains for their respective ride types!

**Comment by ZeeMaji** [comment]
As a cherry on top the classic wooden roller coaster now has rating calculations that are more accurate to it's RCT1 counterpart, so with that it should now be almost fully accurate to how it was in RCT1.
